### DISCUSSION TITLE
failed to run custom build command for `hidapi-rusb v1.3.2` when building on MacOS

### DISCUSSION THREAD

**Discussion by gnkz** [foundry-rs/foundry#6435] [Help]
Hey, I'm trying to build the project using `cargo build` but I'm getting this error:

```
failed to run custom build command for `hidapi-rusb v1.3.2`
```

My rust version is: `rustc 1.74.0 (79e9716c9 2023-11-13)` and I have `libusb` installed (`libusb: stable 1.0.26 (bottled), HEAD`)

Any tips on how can I solve this?

**Comment by CatInTheRain** [comment]
Hi! I use a fresh docker image of `ubuntu:22.04`. My rust version is `rustc 1.76.0 (07dca489a 2024-02-04)`. I had the same error and I installed the following packages:
```bash
sudo apt install build-essential cmake pkg-config libudev-dev
```
and I solved the issue
